README.md

```diff
@@ -208,6 +222,38 @@ res.crystal                             returns int
 res.deuterium                           returns int
 </pre>
 
+### get/set resources settings
+<pre>
+empire.resources_settings(id)                    returns class(object)
+empire.resources_settings(id, settings)          returns class(object)
+</pre>
+
+```python
+current_settings = empire.resources_settings(id)
+
+print(
+       current_settings.list,
+       current_settings.metal_mine,
+       current_settings.crystal_mine,
+       current_settings.deuterium_mine,
+       current_settings.solar_plant,
+       current_settings.fusion_plant,
+       current_settings.solar_satellite,
+       current_settings.crawler
+     )
+
+new_settings = empire.resources_settings(id,
+        settings={
+            "metal_mine": speed.max,
+            "crystal_mine": speed.min,
+            "fusion_plant": 0,
+            "solar_satellite": speed._50,
+        }
```

Comment:
could you make the use of the constants instead of strings. you rely on the dev to memorise and write the correct string. if the string has a misspell it will fail only at runtime

Reply:
Oh, yeah! Actually, I could take a cue from the other functions like **send_fleet**, from the **fleet** parameter. Guess I need to move to an IDE